### PR TITLE
Docs 2.1 Site Architecture Overhaul

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

**Comment by cwarnermm** [comment]
@justinegeffen - I recommend pulling this branch down locally to review the changes for Docs 2.1 scheduled for July 16, rather than trying to review the PR changes via GitHub. 

This is a HUGE PR, but the vast majority of changes are to move source files to their new locations, and to re-wire everything to work as expected. A number of build warnings and errors have also been fixed as part of this PR.

Here's a summary of the changes in this PR
- All active pages now under one of the new site architecture folders: about, comply, configure, deploy, getting-started, incident-collaboration, install, manage, messaging, onboard, scale, and upgrade
- All legacy content now lives under an archive folder. Please review the list of source files in this folder to ensure that I haven't mis-archived anything.
- Running a docs build now ignores all content in the archive folder (via a conf.py exclude-patterns config)
- There are 3 folders that remain outstanding -- I wanted to wait until you're back to take action on the source files within: developer, integrations, and process. Are they all safely moved elsewhere to new homes? If so, we can safely archive these files too.

**Comment by cwarnermm** [comment]
Screen Shot 2021-07-12 at 10 17 50 AM

File: source/manage/command-line-tools.rst [hunk 1]
```diff
@@ -103,19 +103,16 @@ The preceding documentation and command reference below also applies to the `Mat
...
 
-Notes:
-
--  Parameters in CLI commands are order-specific.
--  If special characters (``!``, ``|``, ``(``, ``)``, ``\``, ``'``, and ``"``) are used, the entire argument needs to be surrounded by single quotes (e.g. ``-password 'mypassword!'``, or the individual characters need to be escaped out (e.g. ``-password mypassword\!``).
--  Team name and channel name refer to the handles, not the display names. So in the url ``https://community.mattermost.com/core/channels/town-square`` team name would be ``core`` and channel name would be ``town-square``.
+  -  Parameters in CLI commands are order-specific.
+  -  If special characters (``!``, ``|``, ``(``, ``)``, ``\``, ``'``, and ``"``) are used, the entire argument needs to be surrounded by single quotes (e.g. ``-password 'mypassword!'``, or the individual characters need to be escaped out (e.g. ``-password mypassword\!``).
+  -  Team name and channel name refer to the handles, not the display names. So in the url ``https://community.mattermost.com/core/channels/town-square`` team name would be ``core`` and channel name would be ``town-square``.
```
Comment (justinegeffen) [inline source/manage/command-line-tools.rst:112]:
```suggestion
  -  Team name and channel name refer to the handles, not the display names. So in the URL ``https://community.mattermost.com/core/channels/town-square`` team name would be ``core`` and channel name would be ``town-square``.
```

**Review by justinegeffen — COMMENTED** [review]
I've made some editorial suggestions. Other than that, this looks awesome! Well done!!

**Review by justinegeffen — APPROVED** [review]
Fantastic! :)

**Pull request merged**